1/test/run

```diff
@@ -102,12 +102,16 @@ function test_get_job() {
 }
 
 function run_tests() {
-  local name=$1 ; shift
+  local name=$1
   CONTAINER_ARGS="-u 10000:0"
+  if [ $# -gt 1 ];
+  then
+      CONTAINER_ARGS+=" -e OPENSHIFT_JENKINS_JVM_ARCH=i386 "
+  fi
```

Comment:
we handle this more generically in other test code:
https://github.com/sclorg/mongodb-container/blob/master/3.2/test/run#L163-L167
https://github.com/sclorg/mongodb-container/blob/master/3.2/test/run#L291-L299